pepdbagent/modules/annotation.py

```diff
@@ -0,0 +1,341 @@
+            except RegistryPathError as err:
+                _LOGGER.error(str(err), registry_paths)
+                return AnnotationReturnModel()
+            return self.get(namespace=namespace, name=name, tag=tag, admin=admin)
+
+    def _get_single_annotation(
+        self,
+        namespace: str = None,
+        name: str = None,
+        tag: str = None,
+        admin: Union[List[str], str] = None,
+    ) -> Union[AnnotationModel, None]:
+        """
+        Retrieving project annotation dict by specifying project name
+        :param namespace: project registry_path - will return dict of project annotations
+        :param name: project name in database
+        :param tag: tag of the projects
+        :param admin: list of namespaces where user is admin
+        :return: pydantic Annotation Model of annotations of current project
+        """
+        admin_tuple = tuple_converter(admin)
+        sql_q = f"""
+                select 
+                    {NAMESPACE_COL},
+                    {NAME_COL},
+                    {TAG_COL},
+                    {PRIVATE_COL},
+                    {PROJ_COL}->>'description',
+                    {N_SAMPLES_COL},
+                    {SUBMISSION_DATE_COL},
+                    {LAST_UPDATE_DATE_COL},
+                    {DIGEST_COL}
+                        from {DB_TABLE_NAME}
+                """
+
+        if tag is None:
+            tag = DEFAULT_TAG
+
+        if name:
+            sql_q = f""" {sql_q} where {NAME_COL}=%s and {NAMESPACE_COL}=%s and {TAG_COL}=%s 
+                                and ({PRIVATE_COL} is %s or {NAMESPACE_COL} in %s );"""
+            found_prj = self.con.run_sql_fetchone(
+                sql_q, name, namespace, tag, False, admin_tuple
+            )
+
+        else:
+            _LOGGER.error(
+                "You haven't provided name, digest nor id! Execution is unsuccessful"
+            )
+            _LOGGER.info("Files haven't been downloaded, returning empty dict")
+            return AnnotationModel()
+
+        _LOGGER.info(f"Project has been found!")
+        if len(found_prj) > 0:
+            annot = AnnotationModel(
+                namespace=found_prj[0],
+                name=found_prj[1],
+                tag=found_prj[2],
+                is_private=found_prj[3],
+                description=found_prj[4],
+                number_of_samples=found_prj[5],
+                submission_date=str(found_prj[6]),
+                last_update_date=str(found_prj[7]),
+                digest=found_prj[8],
+            )
+            return annot
+        else:
+            _LOGGER.error(f"Project '{namespace}/{name}:{tag}' was not found.")
+
+        return None
+
+    def _count_projects(
+        self,
+        namespace: str,
+        search_str: str = "",
+        admin: Union[str, List[str]] = False,
+    ) -> int:
+        """
+        Get total number of found projects. [This function is related to _find_projects]
```

Comment:
Instead of "Get total number of found projects.", Just plain "Count projects" is actually more clear...